content/docs/actors/pay_per_result_actors.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Actors paid per result
+description: Learn how actors that are paid per result work. Find out how this differ from the traditional "pay for usage" model and how this benefits you.
```

Comment:
Find out how this model differs from the traditional "pay for usage" model and how this benefits you.